SPECS/mariadb/mariadb.spec

```diff
@@ -1,14 +1,22 @@
 Summary:        Database servers made by the original developers of MySQL.
 Name:           mariadb
-Version:        10.3.28
-Release:        2%{?dist}
+Version:        10.3.32
+Release:        1%{?dist}
 License:        GPLv2 WITH exceptions AND LGPLv2 AND BSD
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 Group:          Applications/Databases
+
+# A buildable mariadb environment needs functioning submodules that do not work from the archive download
```

Comment:
awesome.  Thanks for doing this.  Note that sometimes devs add a script to the folder to make the job easier for the next guy.

Reply:
Good point, fixed now